README.md

```diff
@@ -2215,7 +2215,7 @@ Map entries in Hazelcast are partitioned across the cluster members. Hazelcast c
 
 These benefits do not come for free, please consider the following trade-offs:
 
-- Clients with a Near Cache will have to hold the extra cached data, which increases memory consumption.
+- Clients with a Near Cache will have to hold the extra cached data, which increases their memory consumption.
```

Comment:
Clients with a Near Cache will have to hold the extra cached data, which increases their respective memory consumption.